neuralcompression/optim/_ramp_const_cosine.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import math
+
+from torch.optim import Optimizer
+from torch.optim.lr_scheduler import LambdaLR
+
+
+class RampConstCosineLRSchedule(LambdaLR):
```

Comment:
Did not know about this one : is it proposed in a paper ? (adding citation ?) or is it something that worked well in practice for you ?

Reply:
Just something that worked well in practice. We document it in the appendix. But I think it's sort of a common knowledge thing for NLP at this point.